CMakeLists.txt

```diff
@@ -255,6 +270,10 @@ if(ENABLE_AIRTUNES)
   endif()
 endif()
 
+if(ENABLE_LIBTORRENT)
+  list(APPEND DEP_DEFINES "-DHAS_LIBTORRENT")
+endif()
```

Comment:
This shouldnt be in here. Going forward the definition should be part of the find module using target_compile_definitions.

https://cmake.org/cmake/help/latest/command/target_compile_definitions.html

Functionally this method will be dependant on https://github.com/xbmc/xbmc/pull/23612